### PR TITLE
fix: resend invite

### PR DESCRIPTION
**Pull request by DeannaSharma**

### CODE REVIEW

File: site/pages/api/admin/delete-user.test.ts [hunk 1]
```diff
@@ -127,21 +84,36 @@ describe("delete-user", () => {
...
-                Username: "2f99b92e-a86f-4457-a2dc-923db4781c53",
-                UserAttributes: [
-                    {
-                        Name: "custom:orgId",
-                        Value: "1234",
-                    },
-                ],
-            }),
-        );
+    it("should redirect to /500 if org admin is trying to delete and account and organisation ids do not match", async () => {
```
Comment (alvak1) [inline site/pages/api/admin/delete-user.test.ts:87]:
```suggestion
    it("should redirect to /500 if org admin is trying to delete an account and organisation ids do not match", async () => {
```

File: site/pages/api/admin/resend-invite.api.ts [hunk 2]
```diff
@@ -30,15 +30,24 @@ const resendInvite = async (req: ResendUserApiRequest, res: NextApiResponse): Pr
             throw Error("Insufficient values provided to resend an invite");
         }
 
-        if (validatedBody.data.orgId !== session.orgId) {
-            throw Error("Users can only delete users within the same organisation");
+        if (!session.isSystemAdmin && !session.isOrgAdmin) {
```
Comment (ljones92) [inline site/pages/api/admin/resend-invite.api.ts:33]: Same here

File: site/pages/admin/user-management.page.tsx [hunk 3]
```diff
@@ -141,6 +158,10 @@ const UserManagement = ({ userList, csrfToken }: UserManagementPageProps): React
                                 name: "group",
                                 value: userToResendInvite.userGroup,
                             },
+                            {
+                                name: "orgId",
```
Comment (ljones92) [inline site/pages/admin/user-management.page.tsx:162]: Do we need to pass through the orgId on this page? We should be grabbing it from the session in the case the user is an org admin

File: site/pages/api/admin/resend-invite.test.ts [hunk 4]
```diff
@@ -125,38 +71,43 @@ describe("resend-invite", () => {
...
-                        Value: "user",
-                    },
-                    {
-                        Name: "email",
-                        Value: "dummy.user@gmail.com",
-                    },
-                ],
-            }),
-        );
+    it("should redirect to /500 if org admin is trying to resent an invite and account and organisation ids do not match", async () => {
```
[the remark this answers is not in the document]
Comment (DeannaSharma) [reply site/pages/api/admin/resend-invite.test.ts:74]: done

File: site/pages/api/admin/delete-user.api.ts [hunk 5]
```diff
@@ -30,8 +30,12 @@ const deleteUser = async (req: DeleteUserApiRequest, res: NextApiResponse): Prom
         const userDetails = await getUserDetails(username);
         const formattedUserDetails = user.parse(userDetails);
 
-        if (!session.isSystemAdmin && formattedUserDetails.organisation !== session.orgId) {
-            throw Error("Users can only delete users within the same organisation");
+        if (!session.isSystemAdmin && !session.isOrgAdmin) {
```
[the remark this answers is not in the document]
Comment (DeannaSharma) [reply site/pages/api/admin/delete-user.api.ts:33]: removed check

File: site/pages/api/admin/resend-invite.api.ts (hunk 2 above)
Comment (DeannaSharma) [reply site/pages/api/admin/resend-invite.api.ts:33]: removed check

File: site/pages/admin/user-management.page.tsx (hunk 3 above)
Comment (DeannaSharma) [reply site/pages/admin/user-management.page.tsx:162]: removed

**Review by ljones92 — APPROVED** [review]

**Pull request merged**